### PR TITLE
Profile details update functionality

### PR DESCRIPTION
**Pull request by nanipunepalle**

### CODE REVIEW

File: src/queries/user.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  if (!token) {
+    return
+  }
+
+  const headers = {
+    Authorization: `Bearer ${token}`,
+    'Content-Type': 'application/json',
+  }
+
+  const { data } = await userClient.put<any>('/users/detail', dto, { headers })
```
[the remark this answers is not in the document]
Comment (nanipunepalle) [reply src/queries/user.ts:20]: @mohitkhatri06  I didn't get what you suggested, env file is updated so I have given /users/detail here

**Review by mohitkhatri06 — APPROVED** [review]

**Review by bharathkumarchowdary — CHANGES_REQUESTED** [review]
- [ ] Please include one unit test for the functionality you introduced
- [ ] Also mask the reset password block in user section such that whenever someone introduces reset password, will just enable it again

**Review by mohitkhatri06 — APPROVED** [review]

**Review by mohitkhatri06 — APPROVED** [review]

**Review by bharathkumarchowdary — CHANGES_REQUESTED** [review]
Please move user data download button to user profile. And make user understand the purpose of the button

**Review by mohitkhatri06 — APPROVED** [review]

**Review by bharathkumarchowdary — APPROVED** [review]

**Comment by mohitkhatri06** [comment]
https://github.com/GVR-Secure-Solutions/lekkalu-frontend/pull/449

**Pull request closed without merging**